### ISSUE TITLE
Edit Relationship to Parents

### ISSUE THREAD

**Issue by miqrogroove** [fisharebest/webtrees#3346]
When I add a new child individual to a family, there is a field named "Relationship to parents".

Is there any way to view or edit this relationship after it's created?

**Comment by miqrogroove** [comment]
Example GEDCOM looks like

```
0 @I113@ INDI
1 FAMC @F926@
2 PEDI adopted
```

The closest option I can find is "Link this individual to an existing family as a child".  It's not as clear as an "edit" option.  Is this working as intended?

**Comment by fisharebest** [comment]
It's difficult to fix, due to the way we edit GEDCOM data.

Good news is that I am rewriting all of this.  The new code is in the develop branch.

Once this is done, it will be easy to add.  Perhaps in the "change family members" page?

**Comment by miqrogroove** [comment]
I think FTB does it that way.  There is a "manage children" dialog with both names and relationships.

Also it would make sense to put an edit button on the family tab.  One or both would be nice.  Consider it a low priority.
